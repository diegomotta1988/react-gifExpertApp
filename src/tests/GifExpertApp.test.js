import { shallow } from 'enzyme';
import GifExpertApp from './../GifExpertApp';

describe('pruebas <GifExpertApp /> ', () => {
  test('debe cargar correctamente el componente', () => {
    const wrapper = shallow(<GifExpertApp />);
    expect(wrapper).toMatchSnapshot();
  });

  test('debe mostrar una lista de categorÃ­as', () => {
      const categories = ['Dragon ball, futurama'];
       const wrapper = shallow(<GifExpertApp defaultCategories={categories}/>);
       expect(wrapper.find('GifGrid').length).toBe(categories.length);

      
  })
  
});
