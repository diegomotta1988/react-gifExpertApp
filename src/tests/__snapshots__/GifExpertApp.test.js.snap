// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas <GifExpertApp />  debe cargar correctamente el componente 1`] = `
<Fragment>
  <h2>
    GifExpertApp
  </h2>
  <AddCategory
    setCategories={[Function]}
  />
  <hr />
  <ol />
</Fragment>
`;
